src/main.js

```diff
@@ -106,10 +106,13 @@ var quotes = [
 var savedPosters = [];
 var currentPoster;
 
+
 // event listeners go here 👇
-window.addEventListener('load', generateRandomPoster())
+window.addEventListener('load', generateRandomPoster());
+showRandomButton.addEventListener('click', generateRandomPoster);
```

Comment:
Question for Thursday's Event Listeners class: why didn't we have to include parentheses here but we did with above 'window.'?